doc/guides/modules.rst

```diff
@@ -404,6 +404,35 @@ Module yaml file description
 A module can be described using a file named :file:`zephyr/module.yml`.
 The format of :file:`zephyr/module.yml` is described in the following:
 
+Module name
+===========
+
+Each Zephyr module is given a name by which it can be referred to in the build
+system.
+
+The name is specified in the :file:`zephyr/module.yml` file:
+
+.. code-block:: yaml
+
+   name: <name>
+
+In CMake the location of the Zephyr module can then be referred to using the
+CMake variable ``ZEPHYR_<MODULE_NAME>_MODULE_DIR`` and the variable
+``ZEPHYR_<MODULE_NAME>_CMAKE_DIR`` holds the location of the module's
```

Comment:
"... of the directory containing the module's", for clarity?

Reply:
The clearer the better :+1: